rebuild/config/_icons.scss

```diff
@@ -0,0 +1,152 @@
+    $icon-sizes: ('tiny': 16px, 'small': 24px, 'medium': 32px, 'ui': 40px, 'large': 88px, 'maxi': 152px);
+
+    //Icons and their file locations
+    $icons: (
+        'hamburger--primary': (
+            'asset': 'rebuild/icons/hamburger--primary.svg',
+        ),
+        'hamburger-close--primary': (
+            'asset': 'rebuild/icons/hamburger-close--primary.svg',
+        ),
+        'callback--primary': (
+            'asset': 'rebuild/icons/callback--primary.svg',
+        ),
+        'tooltip--primary-medium': (
+            'asset': 'rebuild/icons/tooltip--primary-medium.svg',
+        ),
+        'tooltip--error': (
+            'asset': 'rebuild/icons/tooltip--error.svg',
+        ),
+        'chevron-left--primary': (
+            'asset': 'rebuild/icons/chevron-left--primary.svg',
+        ),
+        'chevron-right--primary': (
+            'asset': 'rebuild/icons/chevron-right--primary.svg',
+        ),
+        'chevron-down--primary': (
+            'asset': 'rebuild/icons/chevron-down--primary.svg',
+        ),
+        'chevron-right--circular-secondary-solid': (
+            'asset': 'rebuild/icons/chevron-right-circular-secondary-solid.svg',
+        ),
+        'arrow-down--primary': (
+            'asset': 'rebuild/icons/arrow-down--primary.svg',
+        ),
+        'phone--circular-primary': (
+            'asset': 'rebuild/icons/phone--circular--primary.svg',
+        ),
+        'phone--circular-alt-primary-medium': (
+            'asset': 'rebuild/icons/phone--circular--primary-medium.svg',
+        ),
+        'money--primary': (
+            'asset': 'rebuild/icons/money--primary.svg',
+        ),
+        'tick--primary': (
+            'asset': 'rebuild/icons/tick--primary.svg',
+        ),
+        'tick--circular-secondary': (
+            'asset': 'rebuild/icons/tick--circular-secondary.svg',
+        ),
+        'tick--circular-success-solid': (
+            'asset': 'rebuild/icons/tick--circular-success-solid.svg',
+        ),
+        'document--primary': (
+            'asset': 'rebuild/icons/document--primary.svg',
+        ),
+        'document--primary-medium': (
+            'asset': 'rebuild/icons/document--primary-medium.svg',
+        ),
+        'close--primary': (
+            'asset': 'rebuild/icons/close--primary.svg',
+        ),
+        'close--circular-primary-light-solid': (
+            'asset': 'rebuild/icons/close--circular-primary-light-solid.svg',
+        ),
+        'plus--primary': (
+            'asset': 'rebuild/icons/plus--primary.svg',
+        ),
+        'minus--primary': (
+            'asset': 'rebuild/icons/minus--primary.svg',
+        ),
+        'email--primary-medium': (
+            'asset': 'rebuild/icons/email--primary-medium.svg',
+        ),
+        'award--primary-medium': (
+            'asset': 'rebuild/icons/award--primary-medium.svg',
+        ),
+        'service--primary-medium': (
+            'asset': 'rebuild/icons/service--primary-medium.svg',
```

Comment:
I'd maybe call this clock rather than service, think everything else is named for what the icon is rather than its use in the designs?

Reply:
Changed to 'clock' I've checked the others and their names represent the icon design.